tools/data-handler/src/utils/constants.ts

```diff
@@ -0,0 +1,23 @@
+/**
+    Cyberismo
+    Copyright © Cyberismo Ltd and contributors 2024
+
+    This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License version 3 as published by the Free Software Foundation.
+
+    This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public
+    License along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+/**
+ * These are names of the fields that are non-custom fields that present in metadata
+ */
+export const PREDEFINED_FIELDS = [
```

Comment:
Would it somehow be possible to use interface `CardMetadata` (from project-interfaces.ts) and make its members (not the array) content of this array?
That way when we add new members to the card metadata, they would be automatically added here.

Reply:
I'll take a look but I dont think it's possible with interfaces, because they do not exist in the runtime.

Reply:
Ok. Then at least add comment to the interface description that if one expands it, remember to add `PREDEFINED_FIELDS` new value.